docs/guide/install.txt

```diff
@@ -0,0 +1,28 @@
+Installation
+============
+
+Running SQUAD Locally
+---------------------
+SQUAD is a Django application and it is Python3 only. To setup a development environment locally, it can be installed from PyPi using ``pip``:
+
+..code-block:: bash
```

Comment:
does the github syntax work here:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code
It's much more 'techwriter friendly' than rst. That's why I asked about changing to markdown.